scripts/ci/upgrade/proposal.sh

```diff
@@ -8,6 +8,11 @@ apk add -U --no-cache jq tree
 STARGAZE_HOME=/stargaze/starsd
 starsd config keyring-backend test --home $STARGAZE_HOME
 
+echo "BaseApp Params"
+starsd q params subspace baseapp BlockParams --node http://stargaze:26657
```

Comment:
is this gossip protocol?

Reply:
just CI scripts to check the upgrade is doing what  is supposed to do

Reply:
See for example https://ci.publicawesome.com/public-awesome/stargaze/2531/3/16 the max gas params is upgraded to 75M as specified in the upgrade script